src/gracetabnote.ts

```diff
@@ -25,7 +26,7 @@ export class GraceTabNote extends TabNote {
       // grace glyph scale
       scale: 0.6,
       // grace tablature font
-      font: '7.5pt Arial',
+      font: `7.5pt ${Font.SANS_SERIF}`,
```

Comment:
Using the Font.SANS_SERIF or Font.SERIF constant throughout VexFlow allows a developer to change the font family (or specify an entire font stack like `Myriad Pro, Helvetica Neue, Arial, sans-serif`). 

Previously we had Arial hard coded in a lot of places, and we weren't consistent either. Sometimes it said 'sans-serif'. The same happened with Times, Times New Roman, and 'Serif' (yes with the capital letter). I just wanted to make things more consistent throughout our code base.

Reply:
No change required.